index.js

```diff
@@ -8,7 +8,7 @@
 		load: function(callback) {
 			var _this = this;
 
-			this.listeners.push(callback);
+			if (callback) this.listeners.push(callback);
```

Comment:
Please follow the style of the rest of the code. No space after `if` and always use curly braces/blocks even for single line statements

```js
if(callback) {
    this.listeners.push(callback);
}
```